packages/manager/apps/web-office/src/components/layout-helpers/Dashboard/OfficeServiceState.component.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ODS_BADGE_COLOR, ODS_BADGE_SIZE } from '@ovhcloud/ods-components';
+import { OdsBadge } from '@ovhcloud/ods-components/react';
+
+type OfficeServiceStateEnum =
+  | 'autorenewInProgress'
```

Comment:
use an actual enum or object literal

Reply:
It's fixed. Indeed there was an already existing enum type